Dockerfile

```
@@ -0,0 +1,17 @@
+FROM python:3.9
```

Qual tua opinião sobre utilizar uma versão _slim_? 

```suggestion
FROM python:3.9-slim-buster
```